rules/integrations/github/persistence_github_org_owner_added.toml

```diff
@@ -21,7 +21,12 @@ name = "New GitHub Owner Added"
 risk_score = 47
 rule_id = "24401eca-ad0b-4ff9-9431-487a8e183af9"
 severity = "medium"
-tags = ["Domain: Cloud", "Use Case: Threat Detection", "Tactic: Persistence", "Data Source: Github"]
+tags = ["Domain: Cloud", 
```

Comment:
Will bump rule versions, any chance we can add anything else.

Reply:
Will revert the changes

Reply:
Needed to add the new UEBA tag for the threshold rule's detection logic